configure

```diff
@@ -542,7 +545,16 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
-  k = cc_macros()
+  # use 'cc', as CC may be set to a target arch compiler command
+  # in case of cross-compilation
+  if sys.platform.startswith('aix'):
+    k = cc_macros('gcc')
+    matchup = {
+      '_ARCH_PPC'   : 'ppc',
+      '_ARCH_PPC64' : 'ppc64',
+    }
+  else:
+    k = cc_macros('cc')
```

Comment:
I don't think the comment means that you special case cross-compilation but instead avoid issues when you are cross-compiling by using cc in all cases.   I believe if I just remove the comment and the else we'll end up with the same behaviour as before on other platforms.  Should I just go ahead and do that as the person who made the original change may no longer be around so finding the original rational for the non-AIX change may be difficult.